HIP/hip-904.md

```diff
@@ -1131,16 +1128,18 @@ existing accounts. During the integration of this HIP we propose the following:
 - If an account has a one or more `maxAutomaticAssociations` slots, then we can infer that the
   account holder is willing/interested in receiving some unsolicited tokens in their account.
   Therefore, the network will leave this setting to be as it is.
-- If an account has an account with `maxAutomaticAssociations` set to max integer value, then we can
+- If an account has `maxAutomaticAssociations` set to max integer value, then we can
```

Comment:
Honestly I think we should just scrap this bullet point. Since we're not doing a migration this is captured by the previous point.
We can add a sentence to be explicit and say no migration to modify `maxAutomaticAssociations` will be carried out.

Reply:
Updated.